### DISCUSSION TITLE
Grafana 7.5.12 is released but not tagged or added to changelog

### DISCUSSION THREAD

**Discussion by johanmha** [grafana/grafana#43737] [General]
Grafana 7.5.12 is released with a CVE fix as discussed here https://grafana.com/blog/2021/12/10/grafana-8.3.2-and-7.5.12-released-with-moderate-severity-security-fix/, and it may be downloaded here https://grafana.com/grafana/download/7.5.12?platform=linux.

However there is no 7.5.12 tag in the git repo and no 7.5.12 note in the changelog.

What is it that I do not understand?
Thanks for any advice, and also sorry if this is not the right place to ask.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
